lib/game.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require 'frame'
+
+class Game
+  attr_reader :frame, :scores
+
+  TP = 10
+  TF = 10
+  def initialize
+    @scores = []
+    @frame = 0
+  end
+
+  def roll(pins)
+    raise 'Your game is now complete' if game_over?
+
+    @scores << pins
+  end
+
+  def score
```

Comment:
Can this method only be called when the game is over? This is the only way you use it in your tests.
If that is the case, maybe it should give an error if called before the end of the game?
If not, you may want to test partial scores.